urdf/package.xml

```diff
@@ -24,12 +24,12 @@
   <build_depend>console_bridge</build_depend>
   <build_depend>urdfdom</build_depend>
   <build_depend>urdfdom_headers</build_depend>
-  <build_depend>TinyXML</build_depend>
+  <build_depend>tinyxml</build_depend>
   <build_depend>tinyxml_vendor</build_depend>
 
   <exec_depend>urdfdom</exec_depend>
-  <exec_depend>urdfdom-headers</exec_depend>
-  <exec_depend>TinyXML</exec_depend>
+  <exec_depend>urdfdom_headers</exec_depend>
```

Comment:
not sure what we want to do here, usually we use upstream debian package, the rosdep key for it is `liburdfdom-headers-dev`. I think we can use `urdfdom_headers` for now but keep in mind that this should be packaged and be a thirdparty dependency that we install rather than a ros2 package by itself.

Also I'm not sure if we need to depend on both `tinyxml` and `tinyxml_vendor` now that we have a chocolatey package for it. But @nuclearsandwich knows much more about this so I'll leave it to you